harness/determined/searcher/search_runner.py

```diff
@@ -228,6 +236,7 @@ def run(
             logging.info(f"Starting HP searcher for experiment {exp.id}")
             self.search_method.searcher_state.experiment_id = exp.id
             self.search_method.searcher_state.last_event_id = None
+            self.save_state(exp.id)
```

Comment:
We need to save "zero" state. If we do not save it, then on resumption will locate expriment_id file, and terminates since it cannot find folder with searcher state.